src/server/components/dashboard/annualReview/controllers.ts

```diff
@@ -73,21 +74,39 @@ async function confirmNewAnnualReviewDate(req: Request, res: Response): Promise<
   });
 }
 
+function isUpdateKeyDates(list: List) {
+  const currentAnnualReviewStart = list.jsonData.currentAnnualReview?.keyDates.annualReview.START;
+  let shouldUpdate = true;
+  if (currentAnnualReviewStart) {
+    const currentAnnualReviewStartDate = new Date(currentAnnualReviewStart);
+    const today = new Date();
+    if (isEqual(today, currentAnnualReviewStartDate) || isAfter(today, currentAnnualReviewStartDate)) {
+      logger.info(`Annual review already started on ${currentAnnualReviewStartDate.toISOString()}, not updating list.jsonData.currentAnnualReview`);
+      shouldUpdate = false;
+    }
+  }
+  return shouldUpdate;
+}
+
 async function updateNewAnnualReviewDate(req: Request, res: Response): Promise<void> {
   const { id: listId } = res.locals.list;
   const list = (await findListById(listId)) as List;
   const { newAnnualReviewDate } = req.body;
   const annualReviewDate = new Date(newAnnualReviewDate as string);
   const newAnnualReviewDateFormatted = DateFns.format(annualReviewDate, DATE_FORMAT);
 
-  try {
-    await updateAnnualReviewWithKeyDates(list, annualReviewDate.toISOString());
-  } catch (e) {
-    logger.error(e);
-    req.flash("error", "There was a problem updating the annual review date");
-    return res.redirect(`${res.locals.listsEditUrl}/annual-review-date`);
-  }
+  // check if annual review start date has passed and prevent updating if so
+  const isUpdateCurrentAnnualReview = isUpdateKeyDates(list);
 
+  if (isUpdateCurrentAnnualReview) {
```

Comment:
if you decide to take this suggestion 
```
- function isUpdateKeyDates(list: List) {
+ function shouldUpdateKeyDates(list: List) { }
```

you can then just do 
`if(shouldUpdateKeyDates(list)` 

for readability I like to keep function names, parameter names, variables, and all their usage etc all in sync as much as possible. It makes it easier to "follow" to code imo. 

this falls under cognitive complexity. changing/renaming things via local variables or parameters makes code difficult to follow. in this SO answer: 

https://stackoverflow.com/questions/46673399/cognitive-complexity-and-its-effect-on-the-code


>Humans can easily keep in mind about 7 entities +/- 2([wikipedia](https://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two)). When somebody needs to read the code they may run into this limit. Sometimes there are too many local variables to keep track of, or too many if/for statements. In all cases it makes it harder to understand what the code is supposed to do because it's hard to keep a mental picture of the algorithm.